syntax = "proto3";

option go_package = "magma/dp/cloud/go/active_mode_controller/protos/requests";

message RequestPayload {
  string payload = 1;
}

message RequestDbIds {
  repeated int64 ids = 1;
}

message RequestDbId {
  int64 id = 1;
}

message ResponsePayload {
  string payload = 1;
}

service RadioController {
  rpc UploadRequests (RequestPayload) returns (RequestDbIds);
  rpc GetResponse (RequestDbId) returns (ResponsePayload);
}
